mods/coop/hook/lua/ui/lobby/lobbyOptions.lua

```diff
@@ -230,5 +230,27 @@ table.insert(globalOpts,
     }
 )
 
+-- Add the "Common Army" option.
+table.insert(globalOpts,
+    {
+        default = 1,
```

Comment:
Default off please

Reply:
This was done to promote this feature.
If necessary, players can disable this.

Reply:
For now it's gonna stay off by default. Each player with own army. Later if majority of the players prefer this new option, it can change. I'll promote it with the update.
But I won't force such a massive just in the gameplay now.
So change it please.